liip/imagine-bundle/1.8/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Liip\\ImagineBundle\\LiipImagineBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/"
+    },
+    "copy-from-package": {
+    	"Resources/config/routing.yaml": "%ETC_DIR%/routing/imagine.yaml"
```

Comment:
I checked the code of the flex plugin, and the recipe is only taken into account for the *initial* installation of the package. So when updating from a version to the next one, the routing file will **not** be updated (which makes sense as it would overwrite user modifications otherwise, and files added in ETC_DIR are meant to be modified to tweak the config).
So IMO, this is the worse decision maintainers could have taken.

The few users wanting to redefine routes fully can copy/paste the file and use this to replace the import, instead of forcing everyone to copy/paste them on each update.